YandexLogin/YandexLogin/YandexPages/LoginPage.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+
+namespace YandexPageObject.YandexPages
+{
+    public class LoginPage
+    {
+        private WebDriver driver;
+        public LoginPage(WebDriver driver) { this.driver = driver; }
+
+        IWebElement txtUserName => driver.FindElement(By.Id("passp-field-login"));
+        IWebElement btnProceed => driver.FindElement(By.Id("passp:sign-in"));
+        IWebElement txtPassword => driver.FindElement(By.Id("passp-field-passwd"));
+        IWebElement btnLogin => driver.FindElement(By.Id("passp:sign-in"));
+
+        string usernameCreds = "mastermister123";
+        string passwordCreds = "mastermister1231";
+
+        public void InputUsernameAndPassword()
```

Comment:
Make this method as parameterized with the following parameters: username, password. It will allow you to pass above values at the test level which is correct way.

Reply:
Done